doc/RELEASE-0.8.md

```diff
@@ -0,0 +1,50 @@
+
+# Release v0.8
+
+
+We are pleased to announce the release of Screed v0.8. Screed is a database
+engine capable of storing and retrieving short-read sequence data. Screed is
+designed to be fast and adaptable to different sequence file formats. This 
+release of Screed we consider to be stable and complete.
+
+This version of Screed contains developer documentation for contributing to the
+Screed project and a code of conduct for interacting with other contributors
+and project maintainers. Documentation is available at
+http://screed.readthedocs.org/en/v0.8/ 
+
+## Features and Major Changes:
+ - bz2 compressed file streaming
+ - 'accuracy' attribute has been changed to 'quality'
+ - Versioneer version string generator has been added
+ - Dooxygen documentation engine has been added
+ - code of conduct and developer documentation has been added
+ - detailed Changelog has been added
+
+## Closed Issues:
+ - [Screed gzip/bz2 file sniffing]
```

Comment:
Please re-write the names of these to be more user friendly; many of the GitHub issues have opaque titles.

Note that for the khmer release notes I pull out the most important changes under 'New items of note', and I follow up with 'Notable' and 'Additional' bug fixed/issues closed.